### PR TITLE
LPD-28170 - Price list price modifier issues with different currencies

### PR DESCRIPTION
**Pull request by victorcdp**
https://liferay.atlassian.net/browse/LPD-28170

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/pricing/test/CommerceGrossPricingTest.java [hunk 1]
```diff
@@ -934,6 +935,102 @@ public void testPriceModifiersOnProduct() throws Exception {
...
+				ServiceContextTestUtil.getServiceContext());
+
+		CommercePriceList basePriceList =
+			CommercePriceListTestUtil.addCommercePriceList(
+				catalog.getGroupId(), true, 0.0);
+
+		CommercePriceList commercePriceList1 =
+			CommercePriceListTestUtil.addCommercePriceList(
+				catalog.getGroupId(), 0.0);
+
```
Comment (brianikim) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/pricing/test/CommerceGrossPricingTest.java:970]: @victorcdp , could you rename this?

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/pricing/test/CommerceGrossPricingTest.java [hunk 2]
```diff
@@ -934,6 +935,102 @@ public void testPriceModifiersOnProduct() throws Exception {
...
+		_commerceCurrency = commercePriceListCurrency;
+
+		CPInstance cpInstance =
+			CPTestUtil.addCPInstanceWithRandomSkuFromCatalog(
+				catalog.getGroupId());
+
+		CPDefinition cpDefinition = cpInstance.getCPDefinition();
+
+		BigDecimal price1 = BigDecimal.valueOf(50);
+
```
Comment (brianikim) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/pricing/test/CommerceGrossPricingTest.java:989]: Same here. Rename all xxxx1 variables

**Comment by brianikim** [comment]
ci:stop

**Comment by victorcdp** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/pricing/test/CommerceGrossPricingTest.java (hunk 1 above)
Comment (brianikim) [reply modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/pricing/test/CommerceGrossPricingTest.java:970]: @victorcdp , `basePriceList` needs to also be renamed. `baseCommercePriceList`

File: modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/pricing/test/CommerceGrossPricingTest.java [hunk 3]
```diff
@@ -934,6 +935,102 @@ public void testPriceModifiersOnProduct() throws Exception {
...
+			"The price of a product is calculated"
+		).then(
+			"The price modifier is applied to the final converted base price"
+		);
+
+		CommerceCatalog catalog =
+			_commerceCatalogLocalService.addCommerceCatalog(
+				RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+				LocaleUtil.US.getDisplayLanguage(), null,
+				ServiceContextTestUtil.getServiceContext());
```
Comment (brianikim) [inline modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/pricing/test/CommerceGrossPricingTest.java:961]: Same here. it should be `commerceCatalog`.

**Comment by victorcdp** [comment]
ci:stop

**Comment by brianikim** [comment]
ci:stop

**Comment by victorcdp** [comment]
ci:test:relevant

**Comment by victorcdp** [comment]
ci:test:relevant

**Comment by brianikim** [comment]
ci:test:relevant

**Pull request closed without merging**